src/Symfony/Component/Config/ConfigCacheInterface.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config;
+
+/**
+ * Interface for ConfigCache
+ *
+ * @author Matthias Pigulla <mp@webfactory.de>
+ */
+interface ConfigCacheInterface
+{
+    /**
+     * Gets the cache file path.
+     *
+     * @deprecated since 2.7, to be removed in 3.0. Use getPath() instead.
+     * @return string The cache file path
```

Comment:
`@deprecated` should be moved below this one and an empty line should be added between the two directives